lib/Server.js

```diff
@@ -270,8 +270,10 @@ class Server {
     const webpackURL = 'http://' + this.hostname + ':' + this.webpackPort;
     const hot = this.hot;
     const publicPath = hot ? (webpackURL + '/') : null;
+    const platform = this.entry.substring(this.entry.search(/\./) + 1);
```

Comment:
Ideally, the user should be able to name their entry points however they like. I'm thinking we should replace the `--entry` option with `--iosEntry` and `--androidEntry` options.

We'll also probably need to have different webpack 'externals' configs for each platform. Hmm, I wonder if webpack allows different externals for different entry points. If not, we may have to run two webpack servers: one for each platform.